source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,69 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {isEditable} from '../helpers/dom-utils';
+
+const shortcutClass = new Map<string, string>([
+	['e', '.unminimized-comment [aria-label="Edit comment"]:not([hidden])'],
+	['d', '[aria-label="Delete comment"]']
+]);
+
+function runShortcuts(event: KeyboardEvent): void {
+	const focusedComment = select(':target');
+	if (isEditable(event.target) || !focusedComment) {
+		return;
+	}
+
+	if (['j', 'k'].includes(event.key)) {
+		event.preventDefault();
+
+		const items = select.all('.js-minimizable-comment-group')
+			.filter(comment => !comment.querySelector('.minimized-comment:not(.d-none)'));
+		// `j` goes to the next comment, `k` goes back a comment
+		const direction = event.key === 'j' ? 1 : -1;
+
+		const currentIndex = items.indexOf(focusedComment);
+		// Nothing selected or we are on the first comment
+		if (currentIndex === -1 || currentIndex + direction < 0) {
+			return;
+		}
+
+		// Find chosen and clamp it so it cant go past the last one
+		const chosenCommentIndex = Math.min(currentIndex + direction, items.length - 1);
+
+		// Focus comment without pushing to history
+		location.replace('#' + items[chosenCommentIndex].id);
+		// Avoid the extra jump
+		items[chosenCommentIndex].scrollIntoView();
```

Comment:
The first comment works for both

```suggestion
		// Focus comment without pushing to history
		location.replace('#' + items[chosenCommentIndex].id);
		items[chosenCommentIndex].scrollIntoView();
```